.github/workflows/osv-scanner-scheduled.yml

```diff
@@ -85,6 +85,6 @@ jobs:
         # See: https://github.com/github/codeql-action/issues/2117
         actions: read
         contents: read
-    uses: "google/osv-scanner-action/.github/workflows/osv-scanner-reusable.yml@e69cc6c86b31f1e7e23935bbe7031b50e51082de" # ratchet:google/osv-scanner-action/.github/workflows/osv-scanner-reusable.yml@v2.0.2
+    uses: "google/osv-scanner-action/.github/workflows/osv-scanner-reusable.yml@e69cc6c86b31f1e7e23935bbe7031b50e51082de" # ratchet:google/osv-scanner-action/.github/workflows/osv-scanner-reusable.yml@v2.0.3"
     with:
```

Comment:
version should be https://github.com/google/osv-scanner-action/commit/40a8940a65eab1544a6af759e43d936201a131a2

This is the same issue as reported in
https://github.com/erlang/otp/pull/9960#pullrequestreview-2938790196